solutions/swb-reference/README.md

```diff
@@ -6,348 +6,728 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
+
+In the params tab set `projectId` parameter to the `ENV_PROJECT_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+Note: The user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+Send **Stop Environment** request.
+
+PUT `{{API_URL}}/projects/:projectId/environments/:id/stop`
+
+## Start an Environment
+In **SWBv2 Official** Postman Collection under **environments** folder choose **Start Environment** API.
+
+In the params tab set `id` parameter to the `ENV_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+In the params tab set `projectId` parameter to the `ENV_PROJECT_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+Note: The user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+Send **Start Environment** request.
+
+PUT `{{API_URL}}/projects/:projectId/environments/:id/start`
+
+## Terminate the Environment
+In **SWBv2 Official** Postman Collection under **environments** folder choose **Terminate Environment** API.
+
+In the params tab set `id` parameter to the `ENV_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+In the params tab set `projectId` parameter to the `ENV_PROJECT_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+Note: The user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+Send **Terminate Environment** request.
+
+PUT `{{API_URL}}/projects/:projectId/environments/:id/terminate`
+
+# User Management
+In order to create new Admins:
+1. You must go to the Cognito console in your AWS Console.
+1. Under **User pools**, look for and click on `swb-userpool-<stage>-<abbreviation>`.
+1. Under the **Users tab**, choose **Create user**.
+1. Once the user is created, click on the username and under **Group memberships**, choose **Add user to group** to add the user to the Admin group.
+
+## Create Users
+In **SWBv2 Official** Postman Collection under **users** folder choose **Create User** API.
+
+Send **Create User** request.
+
+POST `{{API_URL}}/users`
+
+```json
+{
+    "firstName": "<first name>",
+    "lastName": "<last name>",
+    "email": "<email address>"
+}
+```
+In the response take note of the `id` that was returned. We'll refer to this value as `USER_ID`.
+
+## Assign Project to User
+Note: By default `ITAdmin` users have access permissions to all projects, only `Researchers` and `ProjectAdmin` require project association.
```

Comment:
ITAdmin does _not_ have access permissions to all projects, right? They can list them but not do things within them. I would just have this note say "Only `Researchers` and `ProjectAdmin` require project association."

Reply:
updated to `Only Researchers and ...`